### ISSUE TITLE
Sound doesn't work

### ISSUE THREAD

**Issue by M8is** [M8is/dotfiles#23]
Out of the box the HDMI output is defined as the default sound output. `install` should configure this as well.

**Comment by M8is** [comment]
One solution:
Write to `/etc/asound.conf`:
```
pcm.!default {
    type hw
    card 1
}

ctl.!default {
    type hw           
    card 1
}
```

**Comment by M8is** [comment]
Seems like a bad idea. Instead we should install `pavucontrol`. It's not that big and gives us a GUI for configuring sound devices. Note that it only worked once I got rid of my first suggested change to `/etc/asound.conf`.

**Comment by M8is** [comment]
Fixed in b91fe18f171401bf9fef943d746285b42b2c423c.
